docs/management/admin/event-filters.asciidoc

```diff
@@ -36,12 +36,17 @@ For example, in the KQL search bar, enter the following query to find endpoint n
 --
 +
 [role="screenshot"]
-image::images/event-filter.png[]
+image::images/event-filter.png[Add event filter flyout, 80%]
 . Fill in these fields in the **Details** section:
   .. `Name`: Enter a name for the event filter.
   .. `Description`: Enter a filter description (optional).
 . In the **Conditions** section, depending which page you're using to create the filter, either modify the pre-populated conditions or add new conditions to define how {elastic-sec} will filter events. Use these settings:
   .. `Select operating system`: Select the appropriate operating system.
+  .. Select whether to filter out all matching events or just their descendants: added:[8.15.0,Coming to {serverless-full}.]
+    * `Events`: All matching events are excluded, as well as their descendant events.
+    * `Process Descendants`: Events from the matched process will be ingested, but events from its descendant processes will be excluded. +
+    This option adds the condition `event.category is process` to narrow the filter to process-type events, and then you can add more conditions to identify the process whose descendants you want to exclude.
```

Comment:
This isn't exactly an "apples to apples".  You can have an event filter for non-process events.  I could use an event filter to drop all DNS queries to www.google.com.  Such a filter wouldn't use any `process.*` fields at all.

On the other hand, a "descendent of" filter must describe a process.

```suggestion
  .. Select whether to filter this is a generic event filter which can match any event type, or a filter that describes a process whose descendents' activity will be suppressed: added:[8.15.0,Coming to {serverless-full}.]
    * `Events`: All matching events are excluded. This filter may describe any type of event.
    * `Process Descendants`: Events from the matched process will be ingested, but events from its descendant processes will be excluded. +
    This option adds the condition `event.category is process` to narrow the filter to process-type events.  You can add more conditions to identify the process whose descendants you want to exclude.
```

Reply:
Thanks for the clarification, @gabriellandau! I pushed [a commit](https://github.com/elastic/security-docs/pull/5626/commits/b969eab823c01d6e9cabe52fb0935ff650f094cb) with revisions based on your suggestion; I moved some bits around to streamline the lead-in and put most of the info in the bulleted list items. Please let me know if anything else needs to be adjusted, thanks!